examples/connect/databricks/dash/app.py

```diff
@@ -9,7 +9,11 @@
 from databricks.sdk.core import ApiClient, Config, databricks_cli
 from databricks.sdk.service.iam import CurrentUserAPI
 
-from posit.connect.external.databricks import PositCredentialsStrategy
+from posit.connect.external.databricks import (
+    PositCredentialsStrategy,
+    PositConnectCredentialsStrategy,
+    PositWorkbenchCredentialsStrategy,
```

Comment:
Just a passing observation: it's odd to see `PositWorkbenchCredentialsStrategy` inside the `posit.connect` module. They aren't really relevant for Connect, right? Is it time to create `posit.workbench` to contain these?